src/platform/Kconfig

```diff
@@ -85,7 +85,20 @@ config ICELAKE
 	select CAVS
 	select CAVS_VERSION_2_0
 	help
-	  Select if your target platform is Icelake-compatible
+		Select if your target platform is Icelake-compatible
+
+config ICELAKE_MIN
```

Comment:
This definitely needs a separate patch and proper explanation.

Reply:
Agreed i will send other patch for this

Reply:
removed in this patch